packages/grid/backend/grid/core/config.py

```diff
@@ -15,6 +16,17 @@
 from pydantic import validator
 
 
+def str_to_int(bool_str: Any) -> int:
+    try:
+        return distutils.utils.strtobool(str(bool_str))
+    except ValueError:
+        return 0
+
+
+def str_to_bool(bool_str: Any) -> bool:
+    return bool(str_to_bool(bool_str))
```

Comment:
BUG: I think this would cause an infinite recursion , as we are calling the same function under.

Reply:
good catch. fixed.